specs/lib/gltfSpec.js

```diff
@@ -0,0 +1,363 @@
+'use strict';
+var Cesium = require('cesium');
+var fsExtra = require('fs-extra');
+var path = require('path');
+var Promise = require('bluebird');
+var clone = require('../../lib/clone.js');
+var createGltf = require('../../lib/gltf.js');
+var loadImage = require('../../lib/image.js');
+var loadObj = require('../../lib/obj.js');
+var writeUris = require('../../lib/writeUris.js');
+
+var WebGLConstants = Cesium.WebGLConstants;
+
+var fsExtraReadJson = Promise.promisify(fsExtra.readJson);
+
+var boxObjUrl = 'specs/data/box/box.obj';
+var groupObjUrl = 'specs/data/box-objects-groups-materials/box-objects-groups-materials.obj';
+var boxGltfUrl = 'specs/data/box/box.gltf';
+var groupGltfUrl = 'specs/data/box-objects-groups-materials/box-objects-groups-materials.gltf';
+var diffuseTextureUrl = 'specs/data/box-textured/cesium.png';
+var transparentDiffuseTextureUrl = 'specs/data/box-complex-material/diffuse.png';
+
+describe('gltf', function() {
+    var boxObjData;
+    var groupObjData;
+    var boxGltf;
+    var groupGltf;
+    var diffuseTexture;
+    var transparentDiffuseTexture;
+
+    beforeAll(function(done) {
```

Comment:
Change this to a `beforeEach` and get rid of all of the `clone` usage.  You can then delete the `clone.js`.